package/libyui-qt-doc.spec

```diff
@@ -31,8 +31,9 @@ BuildRequires:  doxygen
 BuildRequires:  fdupes
 BuildRequires:  gcc-c++
 BuildRequires:  graphviz-gnome
-BuildRequires:  libyui-devel >= 3.0.4
 BuildRequires:  texlive-latex
+BuildRequires:  libyui-devel >= 3.4.0
+Requires:       libyui >= 3.4.0
```

Comment:
Is this actually needed? The ncurses part does not include the `libyui` dependency: https://github.com/libyui/libyui-ncurses/pull/60/files#diff-7d1db5a7cdbce3241d4169f4b1dcee22